cellxgene_schema_cli/cellxgene_schema/convert.py

```diff
@@ -1,151 +1,65 @@
 import anndata as ad
-import numpy as np
 
-from . import ontology
-from .remove_labels import AnnDataLabelRemover
+from . import utils
 
 
-def convert(input_file, output_file):
+def convert(input_file, output_file, collection_id, dataset_id):
     print(f"Converting {input_file} into {output_file}")
 
     dataset = ad.read_h5ad(input_file)
 
-    # Rename ethnicity_ontology_term_id field
-    dataset.obs.rename(
-        columns={"ethnicity_ontology_term_id": "self_reported_ethnicity_ontology_term_id"},
-        inplace=True,
-    )
-
-    # Set schema version to 3.0.0
-    dataset.uns["schema_version"] = "3.0.0"
-
-    # Remove Deprecated uns Fields
-    deprecated_uns_fields = [
-        "X_normalization",
-        "default_field",
-        "layer_descriptions",
-        "tags",
-        "version",
-        "contributors",
-        "preprint_doi",
-        "project_description",
-        "project_links",
-        "project_name",
-        "publication_doi",
-    ]
-    for field in deprecated_uns_fields:
-        if field in dataset.uns:
-            del dataset.uns[field]
-
-    # remove labels that are meant to be added by data portal
-    anndata_label_remover = AnnDataLabelRemover(dataset)
-    anndata_label_remover.remove_labels()
-    dataset = anndata_label_remover.adata
-
-    # remove 'ethnicity' label (needs to be done separately since its no longer in the schema definition)
-    if "ethnicity" in dataset.obs:
-        del dataset.obs["ethnicity"]
-
-    # Update deprecated ontologies with known direct replacements
-    disease_ontology_update_map = {
-        "MONDO:0008345": "MONDO:0800029",
-        "MONDO:0004553": "MONDO:0017853",
+    # Set schema version
+    dataset.uns["schema_version"] = "make[1]: Leaving directory '/home/runner/work/single-cell-curation/single-cell-curation'"
```

Comment:
I think @ebezzi might have mentioned this during our call. Strange schema version. 🤓